circle.yml

```diff
@@ -40,5 +40,5 @@ dependencies:
 test:
   override:
     - "gem install danger --version '~> 4.0' && danger"
-    - case $CIRCLE_NODE_INDEX in 0) yarn mocha test/lib && yarn mocha $(find desktop/test -name '*.coffee') && yarn mocha $(find mobile/test -name '*.coffee') ;; 1) yarn mocha $(find desktop/components/*/test -name '*.coffee') && yarn mocha $(find desktop/components/**/*/test -name '*.coffee') ;; 2) yarn mocha $(find desktop/apps/*/test -name '*.coffee') && yarn mocha $(find desktop/apps/*/test -name '*.js') && yarn mocha $(find desktop/apps/*/**/*/test -name '*.coffee') ;; 3) yarn mocha $(find mobile/components/*/test -name '*.coffee') && yarn mocha $(find mobile/components/**/*/test -name '*.coffee') && yarn mocha $(find mobile/apps/*/test -name '*.coffee') && yarn mocha $(find mobile/apps/*/**/*/test -name '*.coffee'); esac:
+    - case $CIRCLE_NODE_INDEX in 0) yarn acceptance && yarn mocha test/lib && yarn mocha $(find desktop/test -name '*.coffee') && yarn mocha $(find mobile/test -name '*.coffee') ;; 1) yarn mocha $(find desktop/components/*/test -name '*.coffee') && yarn mocha $(find desktop/components/**/*/test -name '*.coffee') ;; 2) yarn mocha $(find desktop/apps/*/test -name '*.coffee') && yarn mocha $(find desktop/apps/*/test -name '*.js') && yarn mocha $(find desktop/apps/*/**/*/test -name '*.coffee') ;; 3) yarn mocha $(find mobile/components/*/test -name '*.coffee') && yarn mocha $(find mobile/components/**/*/test -name '*.coffee') && yarn mocha $(find mobile/apps/*/test -name '*.coffee') && yarn mocha $(find mobile/apps/*/**/*/test -name '*.coffee'); esac:
```

Comment:
What do you think about moving these into a bash script since they're also mirrored in `package.json`? I know we just got rid of our `Makefile` setup, but this big test list is getting kinda 🦀 .